app/src/main/java/ch/sdp/vibester/activity/GameActivity.kt

```diff
@@ -179,7 +179,7 @@ open class GameActivity : AppCompatActivity() {
     /**
      * Function to check if the game has ended or not.
      */
-    private fun isEndGame(gameManager: GameManager): Boolean {
+    fun isEndGame(gameManager: GameManager): Boolean {
```

Comment:
Now this is a public function, it would be great to precise the interface in the javadoc.
Meaning the @params and the @return